BedrockCore.cpp

```diff
@@ -42,7 +42,7 @@ bool BedrockCore::peekCommand(BedrockCommand& command) {
                     }
             }
         } catch (const SQLite::timeout_error& e) {
-            SALERT("Command " << command.request.methodLine << " timed out after " << e.time() << "us.");
+            SHMMM("Command " << command.request.methodLine << " timed out after " << e.time() << "us.");
```

Comment:
Wait, but this will silence the webrock timeouts...

Reply:
Yep, Cole said the same thing in the related PR: https://github.com/Expensify/Server-Expensify/pull/2583#issuecomment-375169451
Will see with Tyler what's the best thing to do